pypose/module/lqr.py

```diff
@@ -88,19 +88,104 @@ class LQR(nn.Module):
             \mathbf{c} \left( \mathbf{\tau}_t \right) = \frac{1}{2}
             \mathbf{\tau}_t^\top\mathbf{Q}_t\mathbf{\tau}_t + \mathbf{p}_t^\top\mathbf{\tau}_t
 
+    A discrete-time non-linear system can be described as:
```

Comment:
Let's add some transition here. Maybe a new paragraph title? or some bold text. For the nonlinear system, sometimes people want to solve MPC problem with iterative LQR. Add the citation of this paper https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=dfc73c664f85562956e4b98b3102ebd7e41906ae